rewrite-java/src/main/java/org/openrewrite/java/tree/J.java

```diff
@@ -835,6 +835,10 @@ public Case withStatements(JContainer<Statement> statements) {
         @NonFinal
         transient WeakReference<Padding> padding;
 
+        @Nullable
+        @NonFinal
+        transient WeakReference<Annotations> annotations;
+
```

Comment:
What is this field for?

Reply:
It's like the padding field, giving you access to the `Annotations` class which allows you to access the annotations by position.